testapi.pm

```diff
@@ -1011,9 +1014,10 @@ sub script_run {
     my $cmd  = shift;
     my %args = compat_args(
         {
-            timeout => undef,
-            output  => '',
-            quiet   => testapi::get_var('_QUIET_SCRIPT_CALLS')
+            timeout      => undef,
+            output       => '',
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            max_interval => 250
```

Comment:
```suggestion
            max_interval => undef
```

Reply:
I know, that was just to be sure, now I am pretty sure that my test syntax is still not right